src/testrel.jl

```diff
@@ -517,13 +532,33 @@ function _execute_test(
     engine::String,
     program::String,
     timeout_sec::Int64,
-    readonly::Bool,
+    readonly::Bool;
+    retry_number=1,
 )
     @debug("$name: Starting execution")
-    rsp = exec_async(context, schema, engine, program;
-            readtimeout = timeout_sec, readonly)
+    rsp = try
+        # Exec async really should return after 2-3 seconds
+        exec_async(context, schema, engine, program; readtimeout=30, readonly)
```

Comment:
Just to confirm, was the failure causing the timeouts here? And the `30` now says if we don't return in 30s then we throw an exception?

Reply:
Yeah. The timeouts we have been seeing are here. We used to wait 30 minutes to time out, now we only wait 0.5 minutes. Really, anything over 5 seconds is highly suspicious, but I gave some buffer for network weirdsies.